README.md

```diff
@@ -104,3 +104,41 @@ you may instead play a collection of cards such that:
 - the first card alone would be valid to play
 - the collection includes both a King and Queen
 - the collection optionally includes an additional card if the King and Queen are of the same suit
+
+## Dealer Button
+_Added by Jonathan, 30 Aug 2023_
+
+The Dealer is designated by the Dealer Token.
+If you are Dealer, you may not touch the cards unless you are in possession of the Token.
+
+## Official Cut
+_Added by Matt, 30 Aug 2023_
+
+If you are Dealer, after shuffling, you must designate a Cutter.
+If you are Cutter, you must then receive the Dealer Token, then the deck,
+cut the deck, and return the Token and deck to the Dealer.
+If you are Dealer, you may not thereafter manipulate the cards except to deal them.
+
+## Slam
+_Added by Mathias, 30 Aug 2023_
+
+If you draw a card on your turn instead of playing another move,
+you may play the drawn card if it is otherwise valid to do so.
```

Comment:
In Mathias' phrasing it was more general

On your turn you are allowed to draw a card and then play a card (or a valid set). The turn is assumed to end on a card draw so play fast before the next player takes the turn.